spec/openapi.yaml

```diff
@@ -3927,7 +3927,7 @@ paths:
       description: >-
         Returns a list of issues matching specified filter options. You can
         filter based on the date issues were created, target languages, strings,
-        issue types, states, issue numbers, and which user opened the issue. Unless otherwise
+        issue types, states, issue numbers, job unique ids or not related to any jobs, and which user opened the issue. Unless otherwise
```

Comment:
I'd suggest rewording like:

You can filter based on the date issues were created, strings, target languages, issue types, states, issue numbers, job unique ids, issues not associated with strings in a job, and which user opened the issue.

Reply:
Agree with you, thank you